tests/ui/pages/desktop/search.py

```diff
@@ -62,6 +62,10 @@ def name(self):
 
             def link(self):
                 self.find_element(*self._search_item_name_locator).click()
+                from pages.desktop.details import Detail
```

Comment:
Is this because of circular import errors?

Reply:
Yes